README.md

```diff
@@ -1,7 +1,9 @@
 ## Test262 is the official ECMAScript (ECMA-262) conformance suite.
 
+Test262 tests conformance to the continually maintained draft future ECMAScript standard found at http://tc39.github.io/ecma262/ , together with any Stage 3 or later TC39 proposals. It is maintained by Gorkem Yakin (@goyakin) with significant contributions from many in the ECMAScript community.
+
```

Comment:
I was referring to this: 

> Test262 tests conformance to the continually maintained draft future ECMAScript standard found at http://tc39.github.io/ecma262/ , together with any Stage 3 or later TC39 proposals


I think we have different understanding of how that is actually implemented. The _published_ specification must have tests in which implementers can test against \*. test262 should also contain tests that fit this: 

>  Stage 3 or later TC39 proposals.

However, if those Stage 3 (or later) features are in fact _changes_ to previously published editions of the standard, then it's not clear to me how to proceed. I would expect my assertion above\* to still hold to some degree. Tests for published specification **must** be written and published in this project, then updated to match whatever the new semantics according to the changes defined within the Stage 3 (or later) feature.

As to the maintainer "title", I don't care who is called what, but as a T39 member and someone who is very active in this project, I think it does make sense to also list me as someone that can be contacted for submission review.